packages/emails/src/templates/TeamInviteEmail.tsx

```diff
@@ -22,39 +24,100 @@ export const TeamInviteEmail = (
         team: props.teamName,
         appName: APP_NAME,
       })}>
+      <p style={{ fontSize: "24px", marginBottom: "16px", textAlign: "center" }}>
+        <>{props.language("email_no_user_invite_heading", { appName: APP_NAME })}</>
+      </p>
       <img
-        height="64"
+        style={{
+          borderRadius: "16px",
+          height: "270px",
+          display: "flex",
+          alignItems: "center",
+          justifyContent: "center",
+        }}
         src={
           IS_PRODUCTION
-            ? BASE_URL + "/emails/teamCircle@2x.png"
-            : "http://localhost:3000/emails/teamCircle@2x.png"
+            ? WEBAPP_URL + "/emails/calendar-email-hero.png"
+            : "http://localhost:3000/emails/calendar-email-hero.png"
         }
-        style={{
-          border: "0",
-          display: "block",
-          outline: "none",
-          textDecoration: "none",
-          height: "64px",
-          fontSize: "13px",
-          marginBottom: "24px",
-        }}
-        width="64"
         alt=""
       />
-      <p style={{ fontSize: "24px", marginBottom: "16px" }}>
-        <>
-          {props.language("user_invited_you", {
-            user: props.from,
-            team: props.teamName,
-            appName: APP_NAME,
-          })}
-          !
-        </>
+      <p
+        style={{
+          fontWeight: 400,
+          lineHeight: "24px",
+          marginBottom: "32px",
+          marginTop: "32px",
+          lineHeightStep: "24px",
+        }}>
+        <>{props.language("email_no_user_invite_subheading", { invitedBy: props.from, appName: APP_NAME })}</>
       </p>
-      <p style={{ fontWeight: 400, lineHeight: "24px", marginBottom: "32px" }}>
-        <>{props.language("calcom_explained", { appName: APP_NAME })}</>
+      <div style={{ display: "flex", justifyContent: "center" }}>
+        <CallToAction
+          label={props.language(props.isCalcomMember ? "email_user_cta" : "email_no_user_cta")}
+          href={props.joinLink}
+        />
+      </div>
+      <p
+        style={{
+          fontWeight: 400,
+          lineHeight: "24px",
+          marginBottom: "32px",
+          marginTop: "48px",
+          lineHeightStep: "24px",
+        }}>
+        <>{props.language("email_no_user_invite_steps_intro")}</>
       </p>
-      <CallToAction label={props.language("accept_invitation")} href={props.joinLink} />
+
+      {!props.isCalcomMember && (
+        <div style={{ display: "flex", flexDirection: "column", gap: "32px" }}>
+          <EmailStep
+            translationString={props.language("email_no_user_step_one")}
+            iconsrc={
+              IS_PRODUCTION
+                ? WEBAPP_URL + "/emails/choose-username@2x.png"
+                : "http://localhost:3000/emails/choose-username@2x.png"
```

Comment:
Will this work for self hosted?

Reply:
It should? If webapp_url is set... This is how we handled images in the past in email so if not they wont work either :D